serialized-entity/README.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: pattern
+title: Serialized Entity Pattern
+folder: serialized-entity
+permalink: /patterns/serialized-entity/
+categories: Behavioral
+language: en
+tags:
+- Data access
+---
```

Comment:
Please check the updated YAML frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute